hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegion.java

```diff
@@ -44,31 +44,48 @@ public void close() {
     source.close();
   }
 
-  public void updatePut() {
+  public void updatePut(final long size) {
     source.updatePut();
+    source.updatePutBytes(size);
+    source.updateReceivedBytes(size);
```

Comment:
What is the difference between put and received? They are not the same?

Reply:
Received contains all write operations such as increment/append/put/bulkload, putBytes consists of its own